x/README.md

```diff
@@ -1,6 +1,5 @@
 ---
 sidebar_position: 0
-slug : /modules
```

Comment:
What's the reason of this removal?

Reply:
This is the reason that the `/main/build/modules` routes are broken, the slug isnt allowing the docs to pick up the readme. I removed this on all of the other versions in the sdk-docs repo and it fixed the issue